static/app/views/insights/database/views/databaseSpanSummaryPage.tsx

```diff
@@ -69,6 +79,20 @@ export function DatabaseSpanSummaryPage({params}: Props) {
 
   const sort = decodeSorts(sortField).filter(isAValidSort).at(0) ?? DEFAULT_SORT;
 
+  const {data: indexedSpansByGroupId, isLoading: areIndexedSpansByGroupIdLoading} =
```

Comment:
this api request is needed to get the raw database query on the span page, so that we can then search for issues containing that database query